guides/common/modules/con_major-project-components.adoc

```diff
@@ -0,0 +1,4 @@
+[id="Major-{Project}-Components_{context}"]
+= Major {Project} components
+
+A typical {Project} deployment consists of the following components: a {ProjectServer}, {SmartProxyServers} that mirror content from {ProjectServer}, and managed hosts that receive content from {ProjectServer} and {SmartProxyServers}.
```

Comment:
```suggestion
A typical {Project} deployment consists of the following components: a {ProjectServer}, {SmartProxyServers} that mirror content from {ProjectServer}, and hosts that receive content from {ProjectServer} and {SmartProxyServers}.
```

You only mention "receive content". Maybe also "receive their configuration"? Or what about REX?